upup/models/cloudup/resources/addons/helm-tiller/addon.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Addons
```

Comment:
So I just did a `helm init` on a new cluster and it didn't create an AddOn that I see.  Is the AddOn the right thing to use here?

Thinking out loud here my concern is upgrades to Tiller, we need to be able to upgrade it outside of the cluster lifecycle.  The comments here make me wonder how we can do that: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons#cluster-add-ons.  Specifically the comments are:

1. If an add-on is deleted, it will be recreated automatically.
2. If an add-on is updated through Apiserver, it will be reconfigured to the state given by the supplied fields in the initial config.

Would updating `/etc/kubernetes/addons` be the route forward?